src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -101,6 +101,11 @@ public static void Initialize(Win32PlatformOptions options)
                         new KeyGesture(Key.F10, KeyModifiers.Shift)
                     }
                 })
+                .Bind<KeyGestureFormatInfo>().ToConstant(new KeyGestureFormatInfo(new() 
+                    {
+                        { Key.Back , "Backspace" }, { Key.Down , "Down Arrow" }, { Key.Left , "Left Arrow" },
+                        { Key.Right , "Right Arrow" }, { Key.Up , "Up Arrow" } 
+                    }, meta: "Win"))
```

Comment:
Since on all platforms we have only two variations of key pairs, would it make sense to reuse them similarly to `KeyGestureFormatInfo.Invariant`? Like `FromGeneric("win")` and `FromApple()`.

Just to avoid copy-pasting the same gesture pairs over all non-apple platforms.

Reply:
Possibly, or move all the Generic key pairs to the common overrides dictionary so that only Apple actually needs to specify any.

Reply:
Looks good to me otherwise